compose/envvars-precedence.md

```diff
@@ -5,38 +5,43 @@ keywords: compose, environment, env file
-| 6 | `TAG=$TAG:-1.2`            |  Unset                        |    Unset                          |    `TAG=1.5`   |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.5`           |
-| 7 | `TAG=$TAG:-1.2`            |  Unset                        |    `TAG`                           |    -         |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
-| 8 | `TAG=$TAG:-1.2`            |  Unset                        |    `TAG=1.7`                        |    `TAG`      |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.7`           |
-| 9 | `TAG=$TAG:-1.2`            |  Unset                        |    `TAG=1.7`                        |    `TAG=1.5`   |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.5`           |
-| 10| `TAG=$TAG:-1.2`            |  Unset                        |    `TAG`                           |    `TAG`      |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
-| 11| `TAG=$TAG:-1.2`            |  Unset                        |    `TAG`                           |    `TAG`      |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
-| 12| `TAG=$TAG:-1.2`            |  `TAG=1.8`                    |    -                              |  **`TAG=1.5`** |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.5`           |
-| 13| `TAG=$TAG:-1.2`            |  `TAG=1.8`                    |  **`TAG=1.7`**                      |    `TAG=1.5`   |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.7`           |
-| 14| -                          |**`TAG=1.8`**                  |     -                             |     -        |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.8`           |
-| 15| -                          |  -                            |     `TAG=1.7`                       |     -        |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
-| 16| -                          |  -                            |     `TAG=1.7`                       |     -        |     `TAG=1.6`   |   `TAG=1.4`    | `TAG=1.4`           |
+|  # |  `.env` file      |  OS environment    |    Image    |  `compose.yaml`:`env_file` key  |  `compose.yaml`:`environment` key  |  `run --env`  |  Resolved as  |
+|:--:|:-----------------:|:------------------:|:-----------:|:-------------------------------:|:----------------------------------:|:-------------:|:-------------:|
+|  1 |  `TAG=1.3`        |  `TAG=1.4`         |   -         |   -                             |   -                                |   -           |   -           |
+|  2 |  `TAG=1.3`        |  `TAG=1.4`         |**`TAG=1.5`**|   -                             |   -                                |   -           |  `TAG=1.5`    |
+|  3 |  `TAG=1.3`        |**`TAG=1.4`**       |  `TAG=1.5`  |   -                             |   -                                |**`TAG`**      |  `TAG=1.4`    |
+|  4 |**`TAG=1.3`**      |   -                |  `TAG=1.5`  |**`TAG`**                        |   -                                |   -           |  `TAG=1.3`    |
```

Comment:
wondering if these need spaces between the `|` and the content (`**...`)

Reply:
Not needed. It renders just fine -> https://deploy-preview-15425--docsdocker.netlify.app/compose/envvars-precedence/